diff --git a/include/fcl/narrowphase/detail/traversal/octree/collision/octree_collision_traversal_node.h b/include/fcl/narrowphase/detail/traversal/octree/collision/octree_collision_traversal_node.h
index b1224594..54200820 100644
--- a/include/fcl/narrowphase/detail/traversal/octree/collision/octree_collision_traversal_node.h
+++ b/include/fcl/narrowphase/detail/traversal/octree/collision/octree_collision_traversal_node.h
@@ -39,7 +39,7 @@
 #define FCL_TRAVERSAL_OCTREE_OCTREECOLLISIONTRAVERSALNODE_H
 
 #include "fcl/config.h"
-#if not(FCL_HAVE_OCTOMAP)
+#if !(FCL_HAVE_OCTOMAP)
 #error "This header requires fcl to be compiled with octomap support"
 #endif
 
